main.js

```diff
@@ -0,0 +1,28 @@
+let windowDisplay = document.querySelector(".calc-display")
+console.log(windowDisplay)
+
+let keys = document.querySelectorAll(".button")
+console.log(keys)
+
+let clear = document.querySelector(".clear")
+console.log(clear)
+
+let equal = document.querySelector(".equal")
+
+for (let key of keys){
+    key.addEventListener("click", function() {
+        let newEl = document.createElement("div")
+        let text = document.createTextNode(key.innerText)
+        newEl.appendChild(text)
+        windowDisplay.appendChild(newEl)
+    })
```

Comment:
Hey Danny! I have a suggestion for you that will fix the issue you're having with  the equal operator working. When you create a new TextNode for each of the characters that is clicked, you end up with a string with newline characters in it, and `eval` can't handle that. Instead, here try just appending the innerText from the key to the innerText of the windowDisplay, like this:
```
windowDisplay.innerText += key.innerText
```